src/Expanders.jl

```diff
@@ -491,7 +491,8 @@ function Selectors.runner(::Type{EvalBlocks}, x, page, doc)
     lines = Utilities.find_block_in_file(x.code, page.source)
     cd(page.workdir) do
         result = nothing
-        for (ex, str) in Utilities.parseblock(x.code, doc, page; keywords = false)
+        for (ex, str) in Utilities.parseblock(x.code, doc, page; keywords = false,
+                                              filename="evalblock.jl")
```

Comment:
Couldn't this be named based on the file the block is in, instead of having it be a fixed name?

Reply:
Yea, thought about that, I'll update.

Perhaps that would make sense for `@example`-blocks too? A `@example` block in `index.md` would be in `index.jl` then? Or, perhaps it could actually be `.md`.

Reply:
I think it should be `.md`.

Reply:
Done. Line numbers now also correspond to the source file.